about.html

```diff
@@ -0,0 +1,140 @@
+      <li>
+        <h4>My Page</h4>
+      </li>
+      <li>
+        <h4>Logout</h4>
+      </li>
+    </ul>
+    </div>
+  </div>
+  <header>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"><img src="images/plane3.svg" alt="plane" class="plane"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Back to Home Page</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+  <section class="first-section">
+    <div class="first-section-container">
+      <h1 class="open-heading">The Burning Circle Flight Academy</h1>
+      <div class="first-section-para">
+        <p>
+          The Burining Circle Flight Academy airshow is one of Zambia's first airshows 
+          to be hosted by a privately owned aviation institution. The whole essence of 
+          this event is to connect aerospace industry proffessionals across Africa and
+          the world at Large. This is also a great opportunity for youths interested in
+          aviation and becoming Pilots to learn from the best Pilots and know what it takes
+          to become a world class pilot. 
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions.
+        
+        </p>
+        <p class="email-address">
+          burningcircleflightacademy@gmail.com
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="flex-content">
+      <h2> The Burning Circle Flight Academy Airshow</h2>
+      <hr>
+      <p class="mobile-para">This will be the very first Airshow event to be hosted by Burning circle Flight academy the event will run from November 14th - 18th, so your full particpation and support will be appreciated.</p>
+      <p class='desktop-para'> This will be the very first Airshow event to be hosted by Burning circle Flight academy the event will run from November 14th - 18th, so your full particpation and support will be appreciated.</p>
+    </div>
+    <hr class="horizontal-line">
+      
+  </section>
```

Comment:
- [ ] The middle section in the `About` page is missing the logo/image, please add it so it matches the design.
Design
![image](https://user-images.githubusercontent.com/33767951/141779203-90f53031-7957-4a24-a093-0f9c2d04be0e.png)
About page
![image](https://user-images.githubusercontent.com/33767951/141779261-c32ae897-7548-4db8-b57e-764e9d832ff6.png)

Reply:
done